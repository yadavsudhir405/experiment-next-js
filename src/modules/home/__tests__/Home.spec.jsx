import React from "react"
import { render } from "@testing-library/react"

import { Home } from "../index";

describe('HomeComponent', () => {
    it('should match the snapshot', () => {
        const {container} = render(<Home />);

        expect(container).toMatchSnapshot();
    });
});
