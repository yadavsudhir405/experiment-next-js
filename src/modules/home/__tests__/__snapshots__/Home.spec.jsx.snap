// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeComponent should match the snapshot 1`] = `
<div>
  <h1>
    This is Home page
  </h1>
</div>
`;
